<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BracketedWindow</class>
 <widget class="QWidget" name="BracketedWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Filter Sentences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Filter:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="filterLineEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="listDelegateComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <!-- <widget class="DactListWidget" name="resultsListWidget" native="true"/> -->
    <widget class="QListView" name="resultsList" native="true"/>
   </item>
   <item>
    <widget class="QProgressBar" name="filterProgressBar">
     <property name="visible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <!--
 <customwidgets>
  <customwidget>
   <class>DactListWidget</class>
   <extends>QWidget</extends>
   <header>DactListWidget.hh</header>
  </customwidget>
 </customwidgets>
 -->
 <resources/>
 <connections/>
</ui>
